cloud/aws/variables.tf

```
@@ -86,6 +86,12 @@ variable "create_metric_streams_iam" {
   default     = false
 }
 
+variable "included_services" {
+  description = "List of AWS services to collect metrics for (By default it will collect every AWS service supported by splunk)"
```

We still only use SignalFx instead of Splunk in this repository, so either:
```suggestion
  description = "List of AWS services to collect metrics for. By default, it will collect every AWS service supported by SignalFx"
```
or omit the name:
```suggestion
  description = "List of AWS services to collect metrics for. By default, it will collect every AWS service supported"
```

---

I removed the name.